fairlearn/reductions/_moments/utility_parity.py

```diff
@@ -136,7 +136,7 @@ def load_data(
         This adds a column `event` to the `tags` field.
 
         The `utilities` is a 2-d array which corresponds to g(X,A,Y,h(X))
-        from :footcite:t:`agarwal2018reductions`.
+        from Agarwal et al. (2018).
```

Comment:
why not cite with reference?

Reply:
I had the same question, but then I noticed @TamaraAtanasoska already explained it in the PR description: 

> "Errors caused by referencing the same paper in both a class and a function. Not sure why this shows up, I will open an issue to ask. Right now I have resorted to just referring to the paper hardcoded in the two problematic functions as I couldn't find a better solution."

Reply:
@hildeweerts I will open an issue directly with either the extension or `sphinx`, I need to check what is the best address for it. Maybe I am missing some conf setting or some other way to reference that won't clash, just reading the documentation wasn't telling me anything.